app/scripts/directives/schoollist.coffee

```diff
@@ -20,11 +20,13 @@ angular.module 'edudashApp'
       property: '@property'
       modalLimit: '@modallimit'
       rankby: '=rankby'
+      showHeader: '@showheader'
       school: '=school'
       limit: '=limit'
       sufix: '@sufix'
     link: (scope, el, attrs) ->
       scope.schools = null
+      scope.columns = if attrs.columns? then attrs.columns.split(",") else []
```

Comment:
I think if we put `columns: '=columns'` on `scope:` above, then in `schoollistmodal.html` line 17 it could be an array literal `columns="['REGION', 'DISTRICT']"` instead of parsing out the `,`.

We would still have to deal with setting the default here in `link` though.